source/_disqus.erb

```diff
@@ -1,14 +1,16 @@
-<div class="blog-comments" id="disqus_thread"></div>
-<script type="text/javascript">
-  /* * * CONFIGURATION VARIABLES * * */
-  var disqus_shortname = 'unboxedconsulting';
-  var disqus_identifier = '<%= disqus_identifier %>';
+<section class="blog-comments">
+  <div class="blog-comments__container" id="disqus_thread"></div>
+  <script type="text/javascript">
+    /* * * CONFIGURATION VARIABLES * * */
+    var disqus_shortname = 'unboxedconsulting';
+    var disqus_identifier = '<%= disqus_identifier %>';
 
-  /* * * DON'T EDIT BELOW THIS LINE * * */
-  (function() {
-    var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
-    dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
-    (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
-  })();
-</script>
-<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>
+    /* * * DON'T EDIT BELOW THIS LINE * * */
+    (function() {
+      var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
+      dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
+      (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
+    })();
+  </script>
+  <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>
+</section>
```

Comment:
Disqus inherits the anchor styling from our stylesheets (https://help.disqus.com/customer/portal/articles/545277), but currently we don't style `a` globally.

![screen shot 2015-09-22 at 11 06 29](https://cloud.githubusercontent.com/assets/885223/10015724/03cde460-611a-11e5-81b5-12edc9c942de.png)

Perhaps the blue underline in the Disqus widget should be purple to better match the designs?